packages/client/src/v2-events/features/events/registered-fields/Address.tsx

```diff
@@ -489,12 +492,13 @@ function AddressOutput({ value }: { value?: AddressFieldValue }) {
 }
 
 function useStringifier() {
-  return function useAddressStringifier(value: AddressFieldValue) {
-    /*
-     * As address is just a collection of other form fields, its string formatter just redirects the data back to
-     * form data stringifier so location and other form fields can handle stringifying their own data
-     */
-    const stringifier = useFormDataStringifier()
```

Comment:
This is an anti-pattern. Calling hooks inside higher-order function should be avoided as the returned function could get called conditionally

Reply:
Another location where it's being used this way: https://github.com/opencrvs/opencrvs-core/blob/be1e55301ca68a49c28d71327a454563a691381b/packages/client/src/v2-events/features/drafts/useDrafts.ts#L129-L138